posthog/tasks/calculate_event_property_usage.py

```diff
@@ -13,8 +15,11 @@
 
 @timed("calculate_event_property_usage")
 def calculate_event_property_usage() -> None:
+    tasks = []
     for team_id in Team.objects.values_list("id", flat=True):
-        calculate_event_property_usage_for_team(team_id=team_id)
+        tasks.append(calculate_event_property_usage_for_team_task.s(team_id=team_id))
```

Comment:
then we can append many tasks onto a specific queue (you can specify the queue here too)

now we can keep the queue full and run at a throttled rate with retries

instead of currently where a retry would run for (in this case) 3 to 9 hours